pkg/bundle/setup/clustersetup.sh

```
@@ -17,25 +17,24 @@ PASS_REDHAT="${PASS_REDHAT:-"_PASS_REDHAT_"}"
 MAXIMUM_LOGIN_RETRY="${MAXIMUM_LOGIN_RETRY:-500}"
 
 pr_info() {
-    echo "[INF] $1" | tee -a $LOG_FILE > /dev/null
+    echo "[INF] $1" | tee -a $LOG_FILE >/dev/null
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)